docs/media/S4cal_template.ecf

```diff
@@ -0,0 +1,34 @@
+# Eureka! Control File for Stage 4cal: Calibrated Stellar Spectra
+
+# Stage 4cal Documentation: https://eurekadocs.readthedocs.io/en/latest/ecf.html#stage-4cal
+
+# Transit/Eclipse time
+t0              110.810782      # BJD
+time_offset     60000           # Optional; time offset
+
+# Orbital parameters
+rprs            0.07194         # Planet-star radius ratio
+period          4.2344923       # Orbital period (in Days)
+inc             87.799          # orbital inclination (in degrees)
+ars             12.518          # Semi-major axis
+# Transit durations. When given, the orbital parameters are ignored.
+# t14             0.00            # Total transit duration
+# t23             0.00            # Full transit duration
```

Comment:
Having these set to 0 in the template looks dangerous to me - they only get updated if they're set to `None`. I know these lines are commented out, but still seems safer to either set these to some arbitrary values (like those for WASP-39b or whatever system you've used for the other settings) or set them to `None`